scripts/in_container/run_prepare_provider_readme.sh

```diff
@@ -38,9 +38,10 @@ echo > "${OUT_FILE_PRINTED_ON_ERROR}"
 
 cd "${AIRFLOW_SOURCES}/provider_packages" || exit 1
 
-python3 prepare_provider_packages.py --version-suffix "${TARGET_VERSION_SUFFIX}" \
-    update-package-release-notes "$@"
+PREPARE_PROVIDER_PACKAGES_PY="${AIRFLOW_SOURCES}/dev/provider_packages/prepare_provider_packages.py"
 
+python3 "${PREPARE_PROVIDER_PACKAGES_PY}" --version-suffix "${TARGET_VERSION_SUFFIX}" \
```

Comment:
No single use constants please.

```suggestion
python "${AIRFLOW_SOURCES}/dev/provider_packages/prepare_provider_packages.py" \
    --version-suffix "${TARGET_VERSION_SUFFIX}" \
```

Reply:
They are used above, just the redefine copy&paste was wrong.

Reply:
Not in this file they aren't.

Reply:
Sure, no problem with that.